GliaWidgets/Lib/Coordinator/UIViewControllerCoordinator.swift

```diff
@@ -0,0 +1,89 @@
+import UIKit
+
+class UIViewControllerCoordinator: Coordinator {
+    let id = UUID()
```

Comment:
Let's inject UUID.

Reply:
Fixed in bed95bc30c5eff03a774fed8a670c891b962a7e9